src/cli/commands/gsn-paymaster-fund.ts

```diff
@@ -0,0 +1,36 @@
+import { ether } from '@openzeppelin/test-helpers'
+import program from 'commander'
+import fs from 'fs'
+
+import CommandsLogic from '../CommandsLogic'
+import { Address } from '../../relayclient/types/Aliases'
+import { configureGSN } from '../../relayclient/GSNConfigurator'
+import { gsnCommander } from '../utils'
+
+gsnCommander(['n', 'f', 'h'])
+  .option('--paymaster <address>',
+    'address of the paymaster contract (defaults to address from build/gsn/Paymaster.json if exists')
+  .option('--amount <amount>', 'amount of funds to deposit for the paymaster contract, in wei (defaults to 1 Ether)')
+  .parse(process.argv);
+
+(async () => {
+  const nodeURL = program.ethereumNodeURL !== undefined ? program.ethereumNodeURL : 'http://localhost:8545'
+
+  const relayHubDeployInfo = fs.readFileSync('build/gsn/RelayHub.json').toString()
```

Comment:
this will crash if not found, but you have a default that doesn't use it a little later..

Reply:
I forgot that I created a helper for this... Fixed.